ai-ml/mix-train-and-inference/workloads/fine-tune-l4.yaml

```diff
@@ -0,0 +1,113 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: headless-svc-l4
+spec:
+  clusterIP: None # clusterIP must be None to create a headless service
+  selector:
+    job-name: finetune-gemma-l4 # must match Job name
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: finetune-gemma-l4
+  labels:
+    kueue.x-k8s.io/queue-name: lq
+spec:
+  backoffLimit: 4
+  completions: 2
+  parallelism: 2
+  completionMode: Indexed
+  suspend: true # Set to true to allow Kueue to control the Job when it starts
+  template:
+    metadata:
+      labels:
+        app: finetune-job
+      annotations:
+        gke-gcsfuse/volumes: "true"
+        gke-gcsfuse/memory-limit: "35Gi"
+    spec:
+      priorityClassName: low-priority-preempting
+      containers:
+      - name: gpu-job
+        imagePullPolicy: Always 
+        image: us-docker.pkg.dev/google-samples/containers/gke/gemma-fine-tuning:v1.0.0
```

Comment:
Just a note for future developers (any passerby): This source code for this image exists at https://github.com/GoogleCloudPlatform/accelerated-platforms/tree/974f2eff748d00d2566024d6ec4dd7f309f641c5/use-cases/model-fine-tuning-pipeline/fine-tuning/pytorch/src.